plugins/in_node_exporter_metrics/ne.c

```diff
@@ -1009,9 +1057,15 @@ static struct flb_config_map config_map[] = {
      "scrape interval to collect systemd metrics from the node."
     },
 
+    {
+     FLB_CONFIG_MAP_TIME, "collector.thermalzone.scrape_interval", "0",
```

Comment:
This is just for my curiosity. 
Should we align the name of this thermal_zone related parameter to the actual metrics name?

Reply:
Probably for the best. I'll change it.